index.js

```diff
@@ -23,6 +24,31 @@ const LINK_REGEX = /https:\/\/discord.com\/channels\/(@me|\d{19})\/(\d{18}|\d{19
 const THREAD_TITLE_REGEX = /(?<=\*\*)(.*?)(?=\*\*)/;
 let THREAD_CONFIRMATION_ID = '';
 let ARCHIVED_THREAD = '';
+const MIRROR_USER = process.env.USER1_MARVIN_EMAIL + ', ' + process.env.USER2_MARVIN_EMAIL;
+const usersMap = new Map([
```

Comment:
You're doing great and the logic is solid! Quick pro tip for next: in JS, we usually reach for plain objects `{}` over `Map`, unless we specifically need to preserve insertion order or handle non-string keys

Map is super powerful and it's great that you knew about it and reached for it, just tends to be a bit more niche in everyday usage. So unless if you're optimizing for that specific behavior, Object does it more cleanly

But yeah overall, the code looks good! Your structure and approach is getting sharper each PR!